src/common/reducers/general.reducer.ts

```diff
@@ -67,6 +72,8 @@ export const generalReducer = (state: GeneralState = initialState, action: Gener
     return { ...state, vaultClientLoaded: action.isLoaded };
   case UPDATE_COLLATERAL_TOKEN_BALANCE:
     return { ...state, collateralTokenBalance: action.collateralTokenBalance };
+  case UPDATE_GOVERNANCE_TOKEN_BALANCE:
+    return { ...state, governanceTokenBalance: action.governanceTokenBalance };
```

Comment:
@tomjeatt 
One question about this.
Do we need to manage this via redux store? Actually, we are planning to drop the redux usage at all in this project.

Reply:
Good question and something to think about, but out of scope for this ticket - would be quite a big change and all we're trying to do here is support the governance token ahead of launch.

Reply:
Yes, it makes sense to put it into the redux mechanism for now if we cannot consume the governance balance in the other component tree.